lib/trento/activity_log.ex

```diff
@@ -38,6 +37,12 @@ defmodule Trento.ActivityLog do
     end
   end
 
+  @spec list_activity_log() :: list(ActivityLog.t())
+  def list_activity_log do
+    # This will be made filterable/paginatable in a later PR
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** Avoid comments on future implementations
‏
```suggestion
```

Reply:
My intention here was to motivate/justify the use of `Repo.all` in the following line, which is not very usual. In a way this implementation is partial and I was only drawing attention to this fact.